Files/View Models/SelectedItemsPropertiesViewModel.cs

```diff
@@ -150,6 +150,13 @@ public string ItemMD5Hash
                 }
             }
         }
+        private bool _ItemMD5HashCalcError;
```

Comment:
Remove it from this PR.

Reply:
Sorry but I don't know why my changes on another branch be commit to this branch....

Reply:
Done